packages/features/ee/organizations/components/CreateANewOrganizationForm.tsx

```diff
@@ -93,7 +105,13 @@ const CreateANewOrganizationFormChild = ({ session }: { session: Ensure<SessionC
         handleSubmit={(v) => {
           if (!createOrganizationMutation.isPending) {
             setServerErrorMessage(null);
-            createOrganizationMutation.mutate(v);
+            createOrganizationMutation.mutate({
+              ...v,
+              isPlatform: isPlatformOrg,
+              // we dont have checks to see if an organization url is taken or not if it is a platform user
```

Comment:
I think having to add comment `// we dont have checks to see if an organization url is taken or not if it is a platform user` confirms what I wrote above that maybe its best to have a separate form file for platform.